backend/app/core/db.py

```
@@ -7,15 +7,17 @@
 from app.core.db_factory import create_db_engine
 from app.core.default_prompts import get_default_prompts, get_default_tags
 from app.models import (
-    ProcessingJob,
+    AIConversation,
+    AIResult,
```

Ruff (0.11.9)

`app.models.AIResult` imported but unused

Remove unused import

(F401)